platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminEntry.java

```diff
@@ -127,6 +127,14 @@ public String getPid() {
     return pid;
   }
 
+  public Dictionary<String, Object> getProperties() {
+    return properties;
+  }
+
+  public void setProperties(Dictionary<String, Object> properties) {
```

Comment:
❓ Do we need to expose the whole internal structure? I am not a fan of changing objects to become pojos. Having more specific methods makes it easier to keep track of what is happening. Like a `setProperty()`, `getProperty()`, ...

Reply:
❓ Where are the unit test cases for these new methods?

Reply:
I was only imitating code that was similarly written in master here: https://github.com/codice/ddf/blob/master/platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminEntry.java#L130
I don't think we need to expose the whole internal structure though. If we want to change it, I can update it in both branches.